code/src/components/HappyHeart.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react"
+import moment from 'moment'
+
+export const HappyHeart = ({ id, hearts, date}) => {
+  const [likes, setLikes] = useState(hearts)
+  const [hasLiked, setHasLiked] = useState(false);
+
+  const handleClick = () => {
+    fetch(`https://technigo-thoughts.herokuapp.com/${id}/like`, {
+        method: "POST",
+        body: "",
+        headers: { "Content-Type": "application/json" }
+      }).then(() => {
+        setLikes(likes + 1)
+        setHasLiked(true)
+      })
+    }
+
+return (
```

Comment:
I think this return could need some work. Now it isn't so easy to read...

Reply:
Looks this better?